docs/guides/local-db.mdx

```diff
@@ -0,0 +1,260 @@
+---
+title: "Local Database Setup"
+description: "Set up PostgreSQL for Suga local development."
+---
+
+Suga doesn’t auto-provision databases locally, giving you full choice and control. Here’s how to set up PostgreSQL for local use and any provider (Neon, Supabase, RDS) for production.
+
+<Note>
+  **Before you begin:** Make sure you have Suga CLI installed. See our [Installation Guide](/cli/installation) for platform-specific instructions.
+</Note>
+
+<AccordionGroup>
+  <Accordion title="Step 1: Initialize Project" icon="folder-plus">
```

Comment:
This section should probably be removed and replaced with a link to a getting started guide. That way we won't need to maintain multiple copies of getting started instructions.